### PR TITLE
[#129] Style the monetization user profile pages

### PR DESCRIPTION
**Pull request by shadcn**
This PR adds styles for:

- [x] Team switcher
- [x] Billing Details
- [x] Payment Methods

### Team Switcher
129-team-switcher

### Billing Details
129-billing-details

### Payment Methods
P129-payment-methods

_Note: Looks like the transparency is not working well on the gifs. Pull the PR to test._

### CODE REVIEW

File: themes/custom/apigee_kickstart/templates/user/user--full.html.twig [hunk 1]
```diff
@@ -8,7 +8,10 @@
 {% block content %}
   {# @todo Find a good way to constrict this to col-lg-6 that works with all
```
Comment (jacine) [inline themes/custom/apigee_kickstart/templates/user/user--full.html.twig]: Your CSS fix above completes this todo, so let's remove it. ;)

Comment (shadcn) [reply themes/custom/apigee_kickstart/templates/user/user--full.html.twig]: Done. Thanks.

**Review by jacine — COMMENTED** [review]
This one has conflicts that need to be resolved. Lmk when that's ready.

**Review by jacine — APPROVED** [review]
Awesome!  Thank you @arshad!  LGTM. 👏

**Pull request merged**